tests/test_helpers.py

```diff
@@ -0,0 +1,33 @@
+from django.contrib.contenttypes.models import ContentType
+
+from djangocms_versioning.constants import DRAFT, PUBLISHED
+from djangocms_versioning.models import Version
+from filer.models import File, Folder
+
+from .base import BaseFilerVersioningTestCase
+from djangocms_versioning_filer import helpers
+
+
+class HelperTests(BaseFilerVersioningTestCase):
+
+    def test_check_folder_exists_in_folder(self):
+        self.assertFalse(helpers.check_folder_exists_in_folder(self.folder, 'olivertwist'))
+        subfolder = Folder(
+            parent=self.folder,
+            name='olivertwist'
+        )
+        subfolder.save()
+        self.assertEquals(helpers.check_folder_exists_in_folder(self.folder, 'olivertwist'), subfolder)
```

Comment:
`assertEquals` is deprecated: https://docs.python.org/3/library/unittest.html#deprecated-aliases Should just be `assertEqual`